IdentityCore/src/configuration/webview/MSIDWebviewConfiguration.h

```diff
@@ -26,26 +26,46 @@
 //------------------------------------------------------------------------------
 
 #import <Foundation/Foundation.h>
-#import "MSIDConfiguration.h"
 
 @class MSIDPkce;
-@class MSIDClientInfo;
 
-@interface MSIDWebviewConfiguration : MSIDConfiguration
+@interface MSIDWebviewConfiguration : NSObject
 
 // Common
+@property (readwrite) NSURL *authorizationEndpoint;
+@property (readwrite) NSString *redirectUri;
+@property (readwrite) NSString *clientId;
+@property (readwrite) NSString *resource;
+@property (readwrite) NSOrderedSet<NSString *> *scopes;
+@property (readwrite) NSUUID *correlationId;
 
 @property (readwrite) NSDictionary<NSString *, NSString *> *extraQueryParameters;
+@property (readwrite) NSDictionary<NSString *, NSString *> *sliceParameters;
 @property (readwrite) NSString *promptBehavior;
 @property (readwrite) NSString *claims;
 
-// Is this only for V2?
-@property (readwrite) NSString *requestState;
+// State verifier: Recommended verifier for state value of the response.
+//  Set to YES to stop if verifying state fails
+@property (readwrite) BOOL verifyState;
```

Comment:
nit: I'd prefer verify state to be set on initialize, so nobody can change it in the middle of authentication.

Reply:
sounds logical. will change accordingly.